### ISSUE TITLE
initialize ORM: disk I/O error running migrations, when running docker node on MacOS

### ISSUE THREAD

**Issue by rafarocha** [smartcontractkit/chainlink#1374]
### Problem
I've done all the steps in this [tutorial](https://docs.chain.link/docs/running-a-chainlink-node) from scratch, and more than once, at least in my MacOS, I can not run the local chainlink via docker.

However, I was able to get it to work on [Google Cloud](https://console.cloud.google.com/) through an instance of `Ubuntu 18.04 LTS` with `1 vCPUj`. That is, I was able to connect `Chainlink Node's UI Interface` by navigating to `http://{external-ip-vm-google-cloud}:port`.

Below, I list the respective commands and logs.

### Steps
When I run the procedures the second time, I have removed the folders below, to start literally from scratch.

```
# clear environment's
rm -rf ~/.geth-ropsten
rm -rf ~/.chainlink-ropsten

# remove container and images docker
docker rm {hash_containers}
docker rmi {hash_images}
```

#### 1. To download the docker image and run the [ethereum client geth locally]

```
docker pull ethereum/client-go:stable
mkdir ~/.geth-ropsten
docker run --name eth -p 8546:8546 -v ~/.geth-ropsten:/geth -it \
           ethereum/client-go:stable --testnet --syncmode light --ws \
           --wsaddr 0.0.0.0 --wsorigins="*" --datadir ˜/tmp/geth --ipcdisable
```
**Note**: Additionally, I [needed to include](https://gitter.im/smartcontractkit-chainlink/Lobby?at=5d07b188fbcc305cc488ff52) the `--ipcdisable` option.

#### 2. To download the smartcontract/chainlink: latest image and run it

```
mkdir ~/.chainlink-ropsten

echo "ROOT=/chainlink
LOG_LEVEL=debug
ETH_CHAIN_ID=3
MIN_OUTGOING_CONFIRMATIONS=2
LINK_CONTRACT_ADDRESS=0x20fe562d797a42dcb3399062ae9546cd06f63280
CHAINLINK_TLS_PORT=0
SECURE_COOKIES=false
ALLOW_ORIGINS=*" > ~/.chainlink-ropsten/.env

ETH_CONTAINER_IP=$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' $(docker ps -f name=eth -q))

echo "ETH_URL=ws://$ETH_CONTAINER_IP:8546" >> ~/.chainlink-ropsten/.env
cd ~/.chainlink-ropsten 
docker run -p 6688:6688 -v ~/.chainlink-ropsten:/chainlink -it --env-file=.env smartcontract/chainlink local n
```
And the log after the last command, I got the error below. [Here](https://gist.github.com/rafarocha/99ef5c0cd2a3a770cbbad88b7eaceb00) you can see the complete log!

```
2019-06-18T00:09:39Z [INFO]  Starting Chainlink Node 0.6.0 at commit cba13c9eff4cd15f7ad020df62beb2633e7ee5cb cmd/local_client.go:25  
2019-06-18T00:09:39Z [INFO]  SGX enclave *NOT* loaded                           cmd/enclave.go:11       
2019-06-18T00:09:39Z [INFO]  This version of chainlink was not built with support for SGX tasks cmd/enclave.go:12       
2019-06-18T00:09:39Z [INFO]  Locking sqlite3 for exclusive access with 500ms timeout orm/orm.go:61           
2019-06-18T00:09:39Z [FATAL] Unable to initialize ORM: disk I/O error
error running migrations
github.com/smartcontractkit/chainlink/core/store/migrations.Migrate
	/go/src/github.com/smartcontractkit/chainlink/core/store/migrations/migrate.go:35
github.com/smartcontractkit/chainlink/core/store.initializeORM
```

### Resume
To reproduce the error, only step 2 is required. I described step 1 for didactic purposes. That is, the error happens when trying to start the container node chainlink.

### System Information
- Ethereum client (Geth/Parity) and version: Geth/v1.8.27-stable-4bcc0a37/linux-amd64/go1.11.9
- Go: 1.12.6 darwin/amd64
- Docker: version 17.05.0-ce, build 89658be
- Docker-Machine: 0.16.1, build cce350d
- Host OS: High Sierra 10.13.6
- MacBook Pro (13-inch, Mid 2010)
- 2,4 GHz Intel Core 2 Duo
- 4 GB 1067 MHz DDR3
- NVIDIA GeForce 320M 256 MB

### Environment Variables
```
# node, npm, nvm
export NVM_DIR="$HOME/.nvm"
  [ -s "/usr/local/opt/nvm/nvm.sh" ] && . "/usr/local/opt/nvm/nvm.sh"  # This loads nvm
  [ -s "/usr/local/opt/nvm/etc/bash_completion" ] && . "/usr/local/opt/nvm/etc/bash_completion"
nvm use v10.15.3

# go
GOPATH=/Users/rocha/go
GOBIN=$GOPATH/bin
```

### Additional Information
1. I set the node manager to version 10.15.3, [after updating the development setup guideline](https://github.com/smartcontractkit/chainlink/wiki/Development-Setup-Guide), when I was trying to compile and run the binary locally.
1. I'm using docker through docker-machine due to a [limitation](https://photos.app.goo.gl/cq7fHKeCnLmiSBF68) of modeling and model virtualization that my Macbook has. To successfully use docker on it, I had to perform the following steps, not necessarily in this order.
```
docker-machine create --driver virtualbox default
docker-machine env default
docker-machine start default
eval "$(docker-machine env default)"
```

**Comment by thodges-gh** [comment]
Hey @rafarocha, my initial thought is that the instance doesn't have enough resources to run both an Ethereum client and the Chainlink node. Would you be able to run the Chainlink node on its own instance and see if the I/O error shows up then?

**Comment by rafarocha** [comment]
Run the node as a binary rather than a Docker container, as you [suggested](https://gitter.im/smartcontractkit-chainlink/Lobby?at=5d03ca686f0ec85ade090d69)? 

If it is, at first I did not get it but after setting the version of the node manager to 10.15.3, I did.

However, after I remove project and directories, lower it again and redo the settings, it is taking a long time to complete the build. This time without showing anything in the log.

I have not yet tested [your suggestion](https://gitter.im/smartcontractkit-chainlink/Lobby?at=5d07c31e8e050f62aa352f5a) to use direnv. I will do it.
